evm/vm/logic/call.py

```diff
@@ -244,31 +244,31 @@ class CallEIP150(Call):
     def compute_msg_gas(self, computation, gas, to, value):
         extra_gas = self.compute_msg_extra_gas(computation, gas, to, value)
         return compute_eip150_msg_gas(
-            computation, gas, extra_gas, value, self.mnemonic,
-            constants.GAS_CALLSTIPEND)
+            computation=computation, gas=gas, extra_gas=extra_gas, value=value, mnemonic=self.mnemonic,
```

Comment:
Would you mind formatting these so that they have one parameter per line.

```python
compute_eip150_msg_gas(
    computation=computation,
    gas=gas,
    ...
)
```

Reply:
No problem